docker/templates/default.conf.template

```diff
@@ -19,7 +19,7 @@ server {
         # use INLINE_RUNTIME_CHECK=false yarn build and nonce / hash to apply a CSP
         #add_header Content-Security-Policy "default-src 'none'; script-src 'self'; connect-src 'self'; img-src 'self'; font-src fonts.gstatic.com; style-src 'self' fonts.googleapis.com;" always;
 
-        add_header Referrer-Policy "no-referrer";
+        add_header Referrer-Policy "strict-origin-when-cross-origin";
```

Comment:
Hi @schwartzkylej - I'm loosening this a bit from what you had in 0b9924c6. This is for compatibility with rails origin-based forgery protection. Do you have any concerns with this change? Thanks

Reply:
No that change is fine. Thanks for the heads up.